src/single/scanner.py

```diff
@@ -1,36 +1,49 @@
 #!/usr/bin/env python
+import json
+import os
 import socket
 import subprocess
 import sys
 from datetime import datetime
-import json
 
 # Clear the screen
-subprocess.call('clear', shell=True)
-
-# Pull port range from config.json
-with open("config.json") as json_file:
-    data = json.load(json_file)
-    loRange = int(data['range']['low'])
-    hiRange = int(data['range']['high'])
-
+subprocess.call('cls', shell=True)
 
 # Ask for input
-remoteServer    = input("Enter a remote host to scan: ")
-remoteServerIP  = socket.gethostbyname(remoteServer)
+remoteServer = input("Enter a remote host to scan: ")
+remoteServerIP = socket.gethostbyname(remoteServer)
+
 
 # Print a nice banner with information on which host we are about to scan
-print ("-" * 60)
-print ("Please wait, scanning remote host....", remoteServerIP)
-print ("-" * 60)
+print("-" * 60)
+print("Please wait, scanning remote host....", remoteServerIP)
+print("-" * 60)
+
+
+# Adding config.json path
+def get_absolute_path(relative_path):
+    dir = os.path.dirname(os.path.abspath(__file__))
+    new = os.path.dirname(os.path.dirname(dir))
+    print(new)
+    split_path = relative_path.split("/")
+    absolute_path = os.path.join(new, *split_path)
+    return absolute_path
+
 
 # Check what time the scan started
 t1 = datetime.now()
 
-# scanning the port in range specified in config.json
 
+# Pull port range from config.json
+with open(get_absolute_path('/config.json')) as config_file:
+    config = json.load(config_file)
+    loRange = int(config['range']['low'])
+    hiRange = int(config['range']['high'])
+
+
+# scanning the port in range specified in config.json
 try:
-    for port in range(loRange,hiRange):  
+    for port in range(loRange, hiRange):
```

Comment:
Linting issue. It should be `range(loRange,hiRange):` and not `range(loRange, hiRange): `.
Follow the format of code previously used.